### PR TITLE
Include Standard_L(8|16|32|64)s_v3 into az_l8s_nvme coverage

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
VR: http://autobot.qe.nue2.suse.org/tests/4042 ( Latest )

File: lib/main_publiccloud.pm [hunk 1]
```diff
@@ -55,10 +55,10 @@ sub load_maintenance_publiccloud_tests {
             loadtest "xfstests/generate_report";
         } elsif (get_var('PUBLIC_CLOUD_SMOKETEST')) {
             loadtest "publiccloud/smoketest";
-            loadtest "publiccloud/flavor_check" if (is_ec2());
+            loadtest "publiccloud/flavor_check" if (is_ec2() && !check_var('PUBLIC_CLOUD_SCC_ENDPOINT', 'SUSEConnect'));
```
Comment (grisu48) [inline lib/main_publiccloud.pm:58]: I would add a comment explaining why this is excluded on that particular condition.

**Review by grisu48 — APPROVED** [review]
nitpick otherwise LGTM

File: lib/main_publiccloud.pm (hunk 1 above)
Comment (asmorodskyi) [reply lib/main_publiccloud.pm:60]: done

Comment (asmorodskyi) [reply lib/main_publiccloud.pm:60]: please confirm if text clearly explaining everything

**Comment by asmorodskyi** [comment]
VR : http://autobot.qe.nue2.suse.org/tests/4044 ( maintenance no SUSEConnect )

**Comment by asmorodskyi** [comment]
VR: http://autobot.qe.nue2.suse.org/tests/4045 ( maintenance SUSEConnect defined )

**Pull request merged**